Makefile

```diff
@@ -0,0 +1,48 @@
+
+.PHONY: build test clean docker
+
+MICROSERVICES=build/release/device-coap
+.PHONY: $(MICROSERVICES)
+
+DOCKERS=docker_device_coap_c
+.PHONY: $(DOCKERS)
+
+VERSION=$(shell cat ./VERSION)
+GIT_SHA=$(shell git rev-parse HEAD)
+
+build: ./VERSION ${MICROSERVICES}
+
+build/release/device-coap:
+	    scripts/build.sh
+
+test:
+	    @echo $(MICROSERVICES)
+
+clean:
+	    rm -f $(MICROSERVICES)
+
+./VERSION:
```

Comment:
I tried to build and run the device-coap. I got the below error. Not sure whether the problem is in my environment or not. Just letting you know. Please cross check.

> [device-coap:1612019775:default:ERROR] devsdk_service_new: no version specified

Then as a workaround, I assigned value for VERSION and executed the binary.

Reply:
VERSION file should be autogenerated but it does so based on git tags, of which we did not have any. I've added a v0.1.0 tag and I think this is ok now.

Reply:
Fine Iain..